onnxruntime/core/graph/contrib_ops/quantization_defs.cc

```diff
@@ -131,149 +116,208 @@ Performs element-wise binary {name} on 8 bit data types (with Numpy-style broadc
+          propagateElemTypeFromInputToOutput(ctx, 2, 0);
+
+          if (!hasInputShape(ctx, 0)) return;
+
+          auto& input_shape = getInputShape(ctx, 0);
+          updateOutputShape(ctx, 0, input_shape);
+        }));
+
+static const char* DequantizeLinear_ver1_doc = R"DOC(
 The linear dequantization operator. It consumes a quantized data, a scale, a zero point and computes the full precision data.
 The dequantization formula is y = (x - x_zero_point) * x_scale.
 Scale and zero point must have same shape. They must be either scalar (per tensor) or 1-D tensor (per 'axis').)DOC";
 
-  ONNX_MS_OPERATOR_SET_SCHEMA(DequantizeLinear, 1, OpSchema()
-      .Attr("axis",
-            "The axis along which same quantization parameters are applied. It's optional."
-            "If it's not specified, it means per-tensor quantization and input 'x_scale' and 'x_zero_point' must be scalars."
-            "If it's specified, it means per 'axis' quantization and input 'x_scale' and 'x_zero_point' must be 1-D tensors.",
-            AttributeProto::INT,
-            false)
-      .Input(
-          0,
-          "x",
-          "N-D quantized Input tensor to be de-quantized.",
-          "T1")
-      .Input(
-          1,
-          "x_scale",
-          "Scale for input 'x'. It could be a scalar or a 1-D tensor, which means a per-tensor or per-axis quantization."
-          "If it's a 1-D tensor, its number of elements should be equal to the dimension value of 'axis' dimension of input 'x'.",
-          "T2")
-      .Input(
-          2,
-          "x_zero_point",
-          "Zero point for input 'x'. It could be a scalar or a 1-D tensor, which means a per-tensor or per-axis quantization."
-          "If it's a 1-D tensor, its number of elements should be equal to the dimension value of 'axis' dimension of input 'x'.",
-          "T1")
-      .Output(
-          0,
-          "y",
-          "N-D full precision output tensor. It has same shape as input 'x'.",
-          "T2")
-      .TypeConstraint(
-          "T1",
-          {"tensor(int8)", "tensor(uint8)"},
-          "Constrain 'x' and 'x_zero_point' to 8-bit integer tensors.")
-      .TypeConstraint(
-          "T2",
-          {"tensor(float16)", "tensor(float)"},
-          "Constrain 'y', 'x_scale' to float tensors.")
-      .SetDoc(DequantizeLinear_ver1_doc)
-      .TypeAndShapeInferenceFunction([](ONNX_NAMESPACE::InferenceContext& ctx) {
-        auto y_type = ctx.getOutputType(0);
-        // only float is supported
-        y_type->mutable_tensor_type()->set_elem_type(ONNX_NAMESPACE::TensorProto::FLOAT);
-
-        if (!hasInputShape(ctx, 0))
-          return;
-
-        auto& input_shape = getInputShape(ctx, 0);
-        updateOutputShape(ctx, 0, input_shape);
-      }));
-
-  ONNX_MS_OPERATOR_SET_SCHEMA(ReduceSumInteger, 1, OpSchema()
-      .SetDoc(R"DOC(
+ONNX_MS_OPERATOR_SET_SCHEMA(DequantizeLinear, 1,
```

Comment:
nit: keep style consistent with QuantizeLinear

Reply:
it seems clang-format 1.9.0 enforces that `DequantizeLinear` be in the same line instead of on a new line. I will leave it as is.